packaging/installer/kickstart.sh

```diff
@@ -606,6 +607,8 @@ handle_existing_install() {
         esac
 
         return 0
+      elif [ "${INSTALL_TYPE}" = "unknown" ]; then
+        fatal "We do not support trying to update or claim installations when we cannot determine the install type.\nYou will need to uninstall the existing install using the same method you used to install it to proceed." F0106
```

Comment:
```suggestion
        fatal "We cannot determine the install type.\nTherefore, you can not update or claim installations with the kickstart script.\nYou will need to uninstall the existing install using the same method you used to install it to proceed." F0106
```

Reply:
@kickoke I’m not sure we need to repeat that we cannot determine the install type as that explicitly gets mentioned in output above.

Reply:
@Ferroin So you're saying that [line 581](https://github.com/netdata/netdata/pull/12134/files#diff-2839b074771e3a925573c5483f63f3d30bff140ee8d67ed3802beeebdda1a9feR581) would be displayed before we send out the error message in line 611?

Reply:
It _should_ be in all cases where this message would be displayed.